### ISSUE TITLE
OpenOCD debug hits RCC error handler

### ISSUE THREAD

**Issue by Blargian** [Blargian/sound_localization#3]
When running under debug using OpenOCD we get stuck in the Error_Handler:

```c
if (HAL_RCC_OscConfig(&RCC_OscInitStruct) != HAL_OK)
{
    Error_Handler();
}
```

This [comment](https://github.com/STMicroelectronics/STM32CubeF4/issues/69#issuecomment-1466808735) explains the issue and also [this comment](https://www.eevblog.com/forum/microcontrollers/stm32-clock-gets-modified-when-debugger-is-connected/) which is that the provided config file is setting the clock to use HSI and PLL at 64MHz but CubeMX configuration is using HSE and setting to 200MHz. I think the mismatch causes the issue.

**Comment by Blargian** [comment]
**Solution**

In CLion instead of using the default provided `.cfg` file we can make a modified copy with matching clock settings.

File is located here: `/usr/share/openocd/scripts/board/stm32f746g.cfg`

**Comment by Blargian** [comment]
Closed by #4
